datasetinsights/estimators/faster_rcnn.py

```diff
@@ -242,7 +242,9 @@ def train_one_epoch(
                     f"{intermediate_loss}"
                 )
                 self.writer.add_scalar(
-                    "training/running_loss", intermediate_loss, examples_seen
+                    "training/intermediate_loss",
```

Comment:
Also confused about what you mean by this fixed a bug. This only change the title on tensorboard from `running_loss` to `intermediate_loss`. In this case `intermediate_loss` is a better name.

Reply:
Previously, we have two different losses written into one place, which is "training/running_loss", and the running_loss on tensorboard looks like this:
![Screenshot from 2020-08-05 19-25-54](https://user-images.githubusercontent.com/22140633/89676320-54f60e00-d8a0-11ea-966e-d25a7911268b.png)